### PR TITLE
Add support for #deploy-queue

### PR DESCRIPTION
**Pull request by abtris**
Add support for `deploy-queue`.

@honzajavorek Can you look at PR if refactor is ok?

### CODE REVIEW

File: blackbelt/handle_github.py [hunk 1]
```diff
@@ -282,7 +282,7 @@ def deploy(pr_url):
 
     click.confirm("Ready for deploy! Do you want me to deploy %s as the new version of Apiary?" % merge_info['sha'], abort=True)
 
-    post_message("@here Deploying \"%(title)s\" in 15 seconds" % merge_info)
+    post_message("@here Deploying \"%(title)s\" in 15 seconds" % merge_info, "#deploy-queue")
```
Comment (honzajavorek) [inline blackbelt/handle_github.py]: single quotes

File: blackbelt/handle_github.py [hunk 2]
```diff
@@ -218,7 +218,7 @@ def merge(pr_url):
         'title': pr['title'],
         'comments': pr['comments'],
         'commits': pr['commits']
-    })
+    }, "#deploy-queue")
```
Comment (honzajavorek) [inline blackbelt/handle_github.py]: single quotes

File: blackbelt/deployment.py [hunk 3]
```diff
@@ -7,6 +7,6 @@
 def deploy_staging():
     branch_name = get_current_branch()
 
-    post_message("@here Deploying branch %s to staging" % branch_name)
+    post_message("@here Deploying branch %s to staging" % branch_name, "#deploy-queue")
```
Comment (honzajavorek) [inline blackbelt/deployment.py]: single quotes

**Comment by honzajavorek** [comment]
Looks good! I'm leaving this open in case you'd like to fix the quotes, but otherwise treat this as accepted and feel free to merge it.

**Comment by abtris** [comment]
Using single quotes is performance reason?

**Comment by honzajavorek** [comment]
No, I'm consistency nazi.

**Pull request merged**